src/hooks/useNavbarMenuConfig.ts

```diff
@@ -13,6 +13,11 @@ export const useNavbarMenuConfig = () => {
                 id: 'client-reset-password',
                 title: 'Reset Password',
                 link: '/client-reset-password'
+            },
+            {
+                id: 'company',
```

Comment:
Create a menu, Company, and add submenu -> Add

Reply:
The current navbar does not have provision for adding sub-menu. This will be picked as a separate task [OM-3128](https://hunar.atlassian.net/browse/OM-3128) and will be released with phase 2 of company creation task.